src/main/webapp/WEB-INF/jsp/includes/results.jsp

```diff
@@ -1,163 +1,166 @@
-<%@include file="tags.jsp"%>
+<%@include file="tags.jsp" %>
 <c:set var="biasingProfileCount" value="${model.biasingProfileCount}"/>
 <c:set var="width" value="${100/biasingProfileCount}"/>
 <div id="scratchArea" style="display: none;"></div>
 <table width="100%">
-    <tr>
+  <tr>
 
     <c:forEach begin="0" end="${biasingProfileCount -1}" varStatus="b">
     <c:set var="name" value="results${b.index}"/>
     <c:set var="results" value="${model[name]}"/>
     <td class="recordColumn recordColumn${b.index}" style="padding-right:2px;" valign="top" width="width:${width}%">
 
-    <c:set var="index" value="${b.index}"/>
-    <div class="columnSpecifics">
-    <div class="columnControls">
-        <c:if test="${b.index > 0}">
-        <a href="javascript:;" onclick="removeColumn(${b.index});">delete</a>
-        </c:if>
-        <a href="javascript:;" onclick="addColumn(${b.index})">copy</a>
-    </div>
-    <a href="javascript:;" onclick="showColumnSpecifics()">Show Column Specifics >></a><br>
-    <span class="error"><pre>${matchStrategyErrors[b.index]}</pre></span>
-    <%@include file="debug.jsp"%>
-
-    <fieldset style="display:${cookie.showColumnSpecifics.value ? 'block' : 'none'}">
-    <legend>Relevance & Recall</legend>
-    <input type="text" id="biasing${b.index}" class="biasingInput" value="${results.biasingProfile}" placeholder="Biasing Profile" style="width:220px;">
-    <br>
-    <%@include file="matchStrategy.jsp"%>
-    <%@include file="sort.jsp"%>
-    <%@include file="semantish.jsp"%>
-    <%@include file="wildcard.jsp"%>
-    </fieldset>
-    <fieldset style="display:${cookie.showColumnSpecifics.value ? 'block' : 'none'}">
-    <legend>Personalized Relevance</legend>
-    <%@include file="sessionId.jsp"%>
-    </fieldset>
-    </div>
-
-    <ol id="replacementRow${b.index}" style="display: none">
-
-    </ol>
-
-
-    <ol id="row${b.index}">
-    <%@include file="record.jsp"%>
-    </ol>
-
-    </c:forEach>
+      <c:set var="index" value="${b.index}"/>
+      <div class="columnSpecifics">
+        <div class="columnControls">
+          <c:if test="${b.index > 0}">
+            <a href="javascript:;" onclick="removeColumn(${b.index});">delete</a>
+          </c:if>
+          <a href="javascript:;" onclick="addColumn(${b.index})">copy</a>
+        </div>
+        <a href="javascript:;" onclick="showColumnSpecifics()">Show Column Specifics >></a><br>
+        <span class="error"><pre>${matchStrategyErrors[b.index]}</pre></span>
+        <c:forEach items="${results.warnings}" var="warning">
+          <span class="warnings"><pre>Warning: ${warning}</pre></span>
+        </c:forEach>
+        <%@include file="debug.jsp" %>
+
+        <fieldset style="display:${cookie.showColumnSpecifics.value ? 'block' : 'none'}">
+          <legend>Relevance & Recall</legend>
```

Comment:
that was already like that. i did change it and it had no effect so sure sounds great :)